### PR TITLE
Add Swizzonic

### PR DESCRIPTION
**Pull request by phallobst**
Alexa rank: 142,948

### CODE REVIEW

File: _data/hosting.yml [hunk 1]
```diff
@@ -451,6 +451,13 @@ websites:
       - totp
     doc: https://support.stackpath.com/hc/en-us/articles/360022992631
 
+  - name: switchplus
```
Comment (Jawshy) [inline _data/hosting.yml]:
```suggestion
  - name: Swizzonic (formerly switchplus)
```
They have since changed names.

File: _data/hosting.yml [hunk 2]
```diff
@@ -451,6 +451,13 @@ websites:
...
+    url: https://switchplus.ch/
+    img: switchplus.png
+    tfa:
+      - sms
+    doc: https://switchie.ch/2fa/
```
Comment (Jawshy) [inline _data/hosting.yml]: This link redirects to https://switchplus.ch/
After a brief search, I couldn't find any documentation.

**Review by Jawshy — CHANGES_REQUESTED** [review]
In addition to their name, they have also changed their logo. 

As they also offer domain registration, should they also be listed under the domains category? We've done this with other sites like VentraIP.

File: _data/hosting.yml (hunk 2 above)
Comment (phallobst) [reply _data/hosting.yml]: The link works for me... Strange. How about this one in English?
https://switchie.ch/en/2-factor-authentication-2fa/

File: _data/hosting.yml (hunk 1 above)
Comment (phallobst) [reply _data/hosting.yml]: Do we really want include the former name?

Comment (Jawshy) [reply _data/hosting.yml]: I included their old name for the edge case that people may search the site use that name.

File: _data/hosting.yml (hunk 2 above)
Comment (Jawshy) [reply _data/hosting.yml]: For me, that link also redirects to https://switchplus.ch/
I can read the contents preview [using a search engine](https://duckduckgo.com/?q=2+factor+authentication+site%3Aswitchie.ch&t=ffab&ia=web).

File: _data/hosting.yml (hunk 1 above)
Comment (phallobst) [reply _data/hosting.yml]: The URL is included in the search terms, so using _Swizzonic_ as the name will be sufficient.

File: _data/hosting.yml (hunk 2 above)
Comment (phallobst) [reply _data/hosting.yml]: I can reproduce that with a VPN connection via North America. I'd say we use the link to the English page anyway.

**Comment by phallobst** [comment]
Handling multiple categories for one website is addressed in #4034. I'd say we wait for the result of that discussion and update Swizzonic later if necessary.

**Review by Jawshy — APPROVED** [review]
👍

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**